### ISSUE TITLE
Provisioning should support Beaker's "system type" proprty

### ISSUE THREAD

**Issue by happz** [teemtee/tmt#2356]
In Beaker job XML, one can request a machine of a particular type via `<system_type value="Machine"/>` element. it would be nice if provisioning plugins backed by Beaker - `beaker` and `artemis` for now - could support this filter.

**Comment by jijianwen** [comment]
The default value for "system type" is "Machine". My team has some "Prototype" type machines. We can't make use of those machines if we are not able to set "system type".

**Comment by L8LiLiang** [comment]
Yes, beaker job will abort if run job on Prototype system with job system_type setting to Machine.
In fact, most of our systems are Prototype.
